types/hand-crafted/JitsiConference.d.ts

```diff
@@ -0,0 +1,142 @@
+import { JitsiConferenceEvents } from './JitsiConferenceEvents';
+import JitsiConnection from './JitsiConnection';
+import JitsiTrackError from './JitsiTrackError';
+import JitsiParticipant from './JitsiParticipant';
+import JitsiLocalTrack from './modules/RTC/JitsiLocalTrack';
+import JitsiRemoteTrack from './modules/RTC/JitsiLocalTrack';
+import JitsiTrack from './modules/RTC/JitsiTrack';
+import Transcriber from './modules/transcription/Transcriber';
```

Comment:
I think this needs to be:
```javascript
import Transcriber from './modules/transcription/transcriber';
```
Fixes an issue on linux (case-sensitive filesystem) where it cannot be imported

Reply:
Good spot